CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Deprecated
 
 ### Removed
+- Drop support for PHP 7.2 and 7.3 (#581)
```

Comment:
```suggestion
- Support for PHP 7.2 and 7.3 (#581)
```

It is read: "Removed support for ...", so I suggest to avoid "Drop".

Reply:
It'd prefer the changelog lines to always be worded like good commit message subjects. i.e., starting with a imperative verb like explained here: https://cbea.ms/git-commit/

And the commit message entries should make sense on its own, i.e., they should work without having to add the section heading. (This also is how we have tried to write the changelog entries so far.)